ocs_ci/ocs/benchmark_operator_fio.py

```diff
@@ -181,10 +181,9 @@ def wait_for_wl_to_complete(self):
             expected_status=constants.STATUS_COMPLETED,
         )
         if not sample.wait_for_func_status(result=True):
-            log.error(
-                f"fio-client pod did not move to running state after {self.timeout_completed} sec"
+            raise TimeoutExpiredError(
+                "fio-client pod did not move to running state after {self.timeout_completed} sec"
```

Comment:
I see you removed f here so the variable will not be expanded here, was this intentional?

Reply:
it was a miss. updated in latest commit